docs/general/ad-filtering/create-own-filters.md

```diff
@@ -2387,12 +2387,11 @@ Rules with `$replace` modifier are supported by AdGuard for Windows, Mac, and An
 
 #### **`urltransform`** {#urltransform-modifier}
 
-The `$urltransform` rules allow you to modify the request URL by replacing the text matched by the regular expression.
+The `$urltransform` rules allow you to modify the request's path and/or query by replacing the text matched by a regular expression.
```

Comment:
```suggestion
The `$urltransform' rules allow you to modify the path and/or query of the request by replacing the matched text with a regular expression.
```

Reply:
Fixed the possessive form. I don't agree with the second edit, though: it changes the meaning, making the statement incorrect :)